packages/react/src/components/payStream/paystreamPricing/index.tsx

```diff
@@ -1,5 +1,57 @@
 import React from 'react';
+import { Currency, Paystream } from '@heliofi/common';
 
-const PaystreamPricing = () => <div />;
+import { timeUnitLabels } from '../time-units';
+import { FormikProps } from '../../baseCheckout/constants';
+import { NumberInput, PriceBanner } from '../../../ui-kits';
+import { formatTotalPrice } from '../../baseCheckout/actions';
+import { useHelioProvider } from '../../../providers/helio/HelioContext';
+
+export type PaystreamPricingProps = FormikProps & {
+  activeCurrency: Currency | null;
+  price: number;
```

Comment:
is this decimalAmount ?

Reply:
yes. should I do any changes here?

Reply:
if it does not escalate out of scope please do

Reply:
okay, will do